pandas_datareader/base.py

```diff
@@ -42,13 +42,14 @@ class _BaseReader(object):
     _chunk_size = 1024 * 1024
     _format = 'string'
 
-    def __init__(self, symbols, start=None, end=None,
+    def __init__(self, symbols, start=None, end=None, freq=None,
```

Comment:
It may be that the base datareader should accept all of the valid pandas/datetime freq strings ('H', 'M', 'Q', 'A',...) and the specific modules for wb or oecd etc should then determine what is suitable for the particular data source? For instance World Bank offers monthly data, quarterly data, and annual data for the various data sources.